migrations/20221125080945_add_column_mapping_to_campaign.js

```diff
@@ -0,0 +1,299 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function up(knex) {
+  return knex.schema
+    .alterTable("all_campaign", (table) => {
+      table.text("column_mapping").nullable();
+    })
+    .then(() =>
+      knex.schema.raw(`
+        create or replace view campaign as
+        select
+          id,
+          organization_id,
+          title,
+          description,
+          is_started,
+          due_by,
+          created_at,
+          is_archived,
+          use_dynamic_assignment,
+          logo_image_url,
+          intro_html,
+          primary_color,
+          texting_hours_start,
+          texting_hours_end,
+          timezone,
+          creator_id,
+          is_autoassign_enabled,
+          limit_assignment_to_teams,
+          updated_at,
+          replies_stale_after_minutes,
+          landlines_filtered,
+          external_system_id,
+          is_approved,
+          autosend_status,
+          autosend_user_id,
+          autosend_limit,
+          messaging_service_sid,
+          column_mapping
+        from all_campaign
+        where is_template = false;
+      `)
+    );
+};
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.down = function down(knex) {
+  return knex.schema
+    .raw(
+      `
+      drop view campaign cascade;
+      create view campaign as
+        select
+          id,
+          organization_id,
+          title,
+          description,
+          is_started,
+          due_by,
+          created_at,
+          is_archived,
+          use_dynamic_assignment,
+          logo_image_url,
+          intro_html,
+          primary_color,
+          texting_hours_start,
+          texting_hours_end,
+          timezone,
+          creator_id,
+          is_autoassign_enabled,
+          limit_assignment_to_teams,
+          updated_at,
+          replies_stale_after_minutes,
+          landlines_filtered,
+          external_system_id,
+          is_approved,
+          autosend_status,
+          autosend_user_id,
+          autosend_limit,
+          messaging_service_sid
```

Comment:
Required: Same as above
```suggestion
          messaging_service_sid,
          autosend_limit
```